content/doc/book/managing/plugins.adoc

```diff
@@ -114,17 +114,17 @@ Installs a plugin either from a file, an URL, or from update center.
 
 The Update Center only allows the installation of the most recently released
 version of a plugin. In cases where an older release of the plugin is desired,
-a Jenkins administrator can download an older `.hpi` archive and manually
+a Jenkins administrator can download an older `.jpi` archive and manually
 install that on the Jenkins controller.
 
 ==== From the web UI
 
-Assuming a `.hpi` file has been downloaded, a logged-in Jenkins administrator
+Assuming a `.jpi` file has been downloaded, a logged-in Jenkins administrator
 may upload the file from within the web UI:
 
 . Navigate to the *Manage Jenkins* > *Manage Plugins* page in the web UI.
 . Click on the *Advanced* tab.
-. Choose the `.hpi` file from your system or enter a URL to the `.hpi` file under the *Deploy Plugin* section.
+. Choose the `.hpi` file from your system or enter a URL to the `.jpi` file under the *Deploy Plugin* section.
```

Comment:
Almost certainly wrong (depends on the source of the file), anything not from another Jenkins home directory is very likely `.hpi`.